### ISSUE TITLE
Webpack build failing: "Can't import the named export 'Readable'"

### ISSUE THREAD

**Issue by mcuppi** [bitinn/node-fetch#617]
**Error**
Importing the newest version of node-fetch (v2.4.0) is causing an error when building a project with webpack (4.30.0). This might be breaking many production environments. This is similar to #502.

```
ERROR in ./node_modules/node-fetch/lib/index.mjs 66:23-31
Can't import the named export 'Readable' from non EcmaScript module (only default export is available)
```

**Cause**
I believe the problem is because webpack doesn't resolve .mjs the same way node does. Importing node-fetch and running node v12.0.0 with experimental modules enabled runs without error. However, importing node-fetch using node v8.16.0 with experimental modules does result in the same error:

```
SyntaxError: The requested module does not provide an export named 'Readable'
```

**Fix**

I believe this can be fixed simply by changing the import in blob.js:

```
import { Readable } from 'stream';
```

to

```
import Stream from 'stream'
const Readable = Stream.Readable;
```

I've submitted a pull request for this fix: #618

**Comment by bitinn** [comment]
I will look into it when I am back on my machine. Thx for the PR.

(From my phone)

**Comment by bitinn** [comment]
Fixed and published as 2.4.1 (`npm show node-fetch version` should verify this, not sure what's up with their website.)
